Server/integrations/forgot_password/forgot_password.py

```diff
@@ -174,7 +175,8 @@ def destroy(self, configurationAttributes):
         return True
 
     def getApiVersion(self):
-        return 1
+        # I'm not sure why is 11 and not 2
+        return 11
```

Comment:
@christian-hawk answer it in this commit https://github.com/GluuFederation/oxAuth/commit/528f933a8bf55b272fb6d78af1f93115fbf47060